pkg/network/ebpf/c/protocols/http/buffer.h

```diff
@@ -17,19 +17,35 @@
 #define READ_INTO_USER_BUFFER(name, total_size)                                                                         \
     static __always_inline void read_into_user_buffer_##name(char *dst, char *src) {                                    \
         bpf_memset(dst, 0, total_size);                                                                                 \
-        long ret = bpf_probe_read_user_with_telemetry(dst, total_size, src);                                            \
+        long ret = bpf_probe_read_user(dst, total_size, src);                                                           \
         if (ret >= 0) {                                                                                                 \
             return;                                                                                                     \
         }                                                                                                               \
         const __u64 read_size_until_end_of_page = PAGESIZE - ((__u64)src % PAGESIZE);                                   \
         const __u64 size_to_read = read_size_until_end_of_page < total_size ? read_size_until_end_of_page : total_size; \
-        bpf_probe_read_user_with_telemetry(dst, size_to_read, src);                                                     \
+        bpf_probe_read_user(dst, size_to_read, src);                                                                    \
         return;                                                                                                         \
-    }                                                                                                                   \
+    }
```

Comment:
That change should be documented in the PR description
ps, We can separate the http2 implementation from http, by adding the "bpf probe read" function as another argument